thirdparty/clamav-db/zimbra-clamav-db/rpm/SPECS/clamav-db.spec

```diff
@@ -0,0 +1,29 @@
+Summary:            Inital ClamAV Databases for ClamAV
```

Comment:
Same as the other comment - change 'Inital' to 'Initial' here and on line 14 below.